lib/src/relic_server.dart

```diff
@@ -93,7 +93,8 @@ class RelicServer {
     }
 
     try {
-      final ctx = request.toContext();
+      final ctx = request
+          .toContext(adapterRequest); // adaptor request will be the token
```

Comment:
```suggestion
          .toContext(adapterRequest); // adapter request will be the token
```

Of just name the parameter if this is not clear here?

Reply:
My english sits in the middle of the Atlantic ocean 😄